Gemfile

```diff
@@ -12,6 +12,8 @@ if Gem::Version.new(RUBY_VERSION) < Gem::Version.new('2.2.2')
   gem 'json', '< 2.0'
 end
 
+gem 'addressable', '~> 2.5'
```

Comment:
This isn't a development need - it should only be in the gemspec.  Gemfile pulls in all gemspec requirements, so this is redundant.